src/main/scala/ru/org/codingteam/horta/plugins/htmlreader/HtmlReaderPlugin.scala

```diff
@@ -39,17 +41,20 @@ class HtmlReaderPlugin() extends BasePlugin with CommandProcessor {
               val url = new URL(address)
               url.getProtocol match {
                 case "http" | "https" =>
-                  val connection = url.openConnection().asInstanceOf[HttpURLConnection]
-                  connection.setRequestMethod("GET")
-                  connection.connect()
+                  Future {
+                    val connection = url.openConnection().asInstanceOf[HttpURLConnection]
+                    connection.setRequestMethod("GET")
+                    connection.connect()
 
-                  val code = connection.getResponseCode
-                  val encoding = Option(connection.getContentEncoding).getOrElse("UTF-8")
-                  responseText.append("HTTP: ").append(code).append(", ")
+                    val code = connection.getResponseCode
+                    val encoding = Option(connection.getContentEncoding).getOrElse("UTF-8")
+                    responseText.append("HTTP: ").append(code).append(", ")
 
-                  val doc = Jsoup.parse(Source.fromURL(url)(Codec(encoding)).take(headerSize).mkString)
-                  val title = doc.title()
-                  responseText.append(title)
+                    val doc = Jsoup.parse(Source.fromURL(url)(Codec(encoding)).take(headerSize).mkString)
+                    val title = doc.title()
+                    responseText.append(title)
+                    Protocol.sendResponse(credential.location, credential, responseText.toString())
+                  }
```

Comment:
@ForNeVeR, looks like Mr IngvarJackal fixed that remark almost a month ago. Could please take a look into that again?

@IngvarJackal just a general suggestion. When you are done with the your changes please reassign the PR to the reviewer and drop a "done" comment. :)

Thanks!